### PR TITLE
Sticky

### PR DESCRIPTION
**Pull request by XavierJp**
A bit of this a bit of that:

- add a sticky header
- update lateral scroll behaviour as sticky header breaks it
- better font loading (font swap)
- better constants management
- add a highlight area for API access section
- add a visit API site button under menu
- add prettierrc and lintrc

NB: About font loading I am still not satisfied. And I fill like we are loading italic and bold italic for nothing.

### CODE REVIEW

**Comment by XavierJp** [comment]
Not ready for merge :

- button hover is weird
- blue highlight area on access section might be too light

File: components/api/section.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import Link from 'next/link';
+
+import globals from '../../const';
 
 const Section = ({ id, title, children }) => {
   return (
-    <section id={id}>
+    <section>
+      <a className="hidden-anchor" id={id}>
+        HEY
```
Comment (rdubigny) [inline components/api/section.js]: what is the point of this ?

File: components/api/support.js [hunk 2]
```diff
@@ -13,31 +14,34 @@ const Support = ({ contact_description, contact_link }) => {
 
         {contact_link ? (
           <>
-            {contact_link.includes("@") ? (
+            {contact_link.includes('@') ? (
```
Comment (rdubigny) [inline components/api/support.js]: We must stop this kind of magic in the future. We must opt for a more declarative approach : use 2 fields `contact_email` & `contact_link`, the FD is then free to use one of them or the both.

File: const/links.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export default {
+  MAIL_REQUEST_API:
```
Comment (rdubigny) [inline const/links.js]: MAIL_REQUEST_API => REQUEST_API_MAILTO_LINK

File: const/links.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    'mailto:contact@api.gouv.fr?subject=Demande%20d%27une%20nouvelle%20API',
+  ADD_API:
```
Comment (rdubigny) [inline const/links.js]: ADD_API_LINK

File: const/links.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+    'https://github.com/betagouv/api.gouv.fr/blob/master/CONTRIBUTING.md#ajouter-une-api',
+  SIGNUP: 'https://signup.api.gouv.fr',
```
Comment (rdubigny) [inline const/links.js]: SIGNUP_LINK

File: components/api/section.js (hunk 1 above)
Comment (XavierJp) [reply components/api/section.js]: With a sticky header anchors get under header. I avoid this using fake anchors. "HEY" I totaly useless here ^^my bad

File: components/api/support.js (hunk 2 above)
Comment (XavierJp) [reply components/api/support.js]: Noted

File: const/links.js (hunk 3 above)
Comment (XavierJp) [reply const/links.js]: OK

File: const/links.js (hunk 4 above)
Comment (XavierJp) [reply const/links.js]: OK

File: const/links.js (hunk 5 above)
Comment (XavierJp) [reply const/links.js]: Ok

**Pull request merged**